### PR TITLE
[Task] Remove `adminer` as Database Explorer

### LINKED ISSUE #12520: [Task] Remove Database Explorer
Since we're going away from adminer it doesn't make sense to move it to a bundle, we just remove it.

### PR DESCRIPTION
**Pull request by kingjia90**
## Changes in this pull request  
Resolves #12520

### CODE REVIEW

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ server {
 
     # Some Admin Modules need this:
     # Database Admin, Server Info
-    location ~* ^/admin/(adminer|external) {
+    location ~* ^/admin/external {
```
Comment (jdreesen) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md:112]: This probably needs to be adapted in the skeleton/demo, too.

Comment (dvesh3) [reply doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md:112]: yes, here https://github.com/pimcore/demo/blob/11.x/.docker/nginx.conf#L94 & https://github.com/pimcore/skeleton/blob/11.x/.docker/nginx.conf#L94

**Review by aashan10 — APPROVED** [review]

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md [hunk 2]
```diff
@@ -109,7 +109,7 @@ server {
 
     # Some Admin Modules need this:
     # Database Admin, Server Info
```
Comment (blankse) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md]:
```suggestion
    # Server Info, Opcache
```

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md [hunk 3]
```diff
@@ -393,7 +393,7 @@ server {
...
```
Comment (blankse) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md]:
```suggestion
    # Server Info, Opcache
```

**Comment by cancan101** [comment]
Is there any suggested replacement for `adminer` and / or was it ever [factored out as `pimcore/database-explorer-bundle`](https://github.com/pimcore/pimcore/issues/12520#issuecomment-1170918451)?

**Comment by kingjia90** [comment]
Officially there no any other replacement for this, at least not as easily installable and integrated bundle.
Maybe just go for phpMyAdmin

**Comment by brusch** [comment]
@cancan101 Adminer is EOL is not longer maintained, see: https://github.com/vrana/adminer/ 
If you're working with a Docker environment, see e.g.: 
https://hub.docker.com/_/phpmyadmin

**Comment by alexthunder3** [comment]
@brusch what about https://github.com/pematon/adminer ? The maintainer has been responding to and fixing the issues that people report in vrana/adminer, plus doing modernization work. Has this been taken into consideration? 
or https://github.com/adminerevo/adminerevo ?

**Comment by kingjia90** [comment]
> @brusch what about https://github.com/pematon/adminer ? The maintainer has been responding to and fixing the issues that people report in vrana/adminer, plus doing modernization work. Has this been taken into consideration? or https://github.com/adminerevo/adminerevo ?

Thank you @alexthunder3 ! 
That's a nice input, but i would like to suggest to open a brand new discussion/issue to gain more visibility and support on this topic.

**Comment by jdreesen** [comment]
Note that `adminerevo/adminerevo` has just been discontinued and archived (and suggests [pematon/adminer](https://github.com/pematon/adminer) as a replacement).

See: https://github.com/adminerevo/adminerevo/discussions/233

**Comment by kingjia90** [comment]
> Note that `adminerevo/adminerevo` has just been discontinued and archived (and suggests [pematon/adminer](https://github.com/pematon/adminer) as a replacement).
> 
> See: [adminerevo/adminerevo#233](https://github.com/adminerevo/adminerevo/discussions/233)

wow.. that announcement was exactly made just 1 hour ago 😮 such a coincidence and timing!

**Comment by brusch** [comment]
@alexthunder3 shouldn't be to hard to create a community bundle out of it 😊

**Comment by jdreesen** [comment]
Also seems like `pematon/adminer` will be moved/renamed: https://github.com/pematon/adminer/discussions/13

**Pull request merged**